package/CHANGELOG

```diff
@@ -16,7 +16,7 @@ The rules for this file:
 -------------------------------------------------------------------------------
 ??/??/?? IAlibay, HeetVekariya, marinegor, lilyminium, RMeli,
          ljwoods2, aditya292002, pstaerk, PicoCentauri, BFedder,
-         tyler.je.reddy
+         SampurnaM,orbeckst,tyler.je.reddy
```

Comment:
```suggestion
         tyler.je.reddy, SampurnaM, orbeckst
```
@orbeckst did you want to be added to the changelog?

Reply:
no, I didn't do anything here
```suggestion
         tyler.je.reddy, SampurnaM
```

Reply:
But, but, you told me how to update the logic and how to make the functions cleaner with suggested code, that I simply copy pasted?